Scripts/renderToDom.js

```diff
@@ -0,0 +1,6 @@
+const renderToDom = (divId, textToRender) => {
```

Comment:
Looks like renderToDom.js file (this file) is not linked to html.

Reply:
not yet, but i figured we'd do more of that tomorrow.